arduino/examples/ESP32IOBoard/ESP32IOBoard.ino

```diff
@@ -215,30 +299,45 @@ void setup() {
     // have performance impact.
     // openmrn.stack()->print_all_packets();
 
-    // Start the TCP/IP listener
-    openMRNServer.setNoDelay(true);
-    openMRNServer.begin();
+#if defined(USE_CAN)
+    // When using the hardware can device it is recommended to also utilize the
```

Comment:
no, it is most definitely not recommended.

Reply:
The problem with not calling the by task is that loop() runs at priority 0 and rx/tx at ~20. Loop() will be called much less often in this case, by using the BG task the OpenMRN loopethod will be called as often or more than RX task.

Reply:
I'll drop the BG task and do some further testing, if it is stable (likely) and not starving the rest of the stack of cycles (some possibility) then the BG task can go away.